docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,70 @@
+
+- Code reviews
+- Unit testing
+- Integration testing
+
+## Software validation
+
+In order to support the repeatability of the validation, the following must be registered:
+
+- The person executing the test
+- The person evaluating the test
+- Date of the test
+- The software tested, including its version number
+- If applicable: configuration, hardware, testing tools
+- The test, including steps to take the test, expected results and acceptance criteria
+- The results, including samples used
+- The evaluation of the test
+- Conclusion of the test
+
+The person executing the test should be independent from the person who developed the software and wrote the validation. Software validation is preferably executed by an end user. For example, someone from the laboratory when the end result is mostly used by the laboratory.
+
+ISO-15189 states that the validation shall be as extensive as is necessary and confirm, through the provision of objective evidence in the form of performance specifications, that the specific requirements for the intended use of the examination have been fulfilled. This indicates that depending on the amount of changes of the new release, the validation could be any length. Sometimes a single note could be enough. For example, for a hotfix where the paths are now correctly stated, a note saying "tested in the first production run, the result files are now saved in the corrected path". Or “Only an integration test has been performed due to a change of a parameter. There were no issues with the first analysis”. There should always be a record of the validation.
+
+Depending on the changes in the software, and in particular for the first version of the software a performance evaluation is needed. MDCG 2020-1
+(Guidance on Clinical Evaluation (MDR) / Performance Evaluation (IVDR) of Medical Device Software) could help in determining the extend of the validation and/or the number of samples needed.
+In particular:
+
+> Sufficient amount
+>
+> - Does the data support the intended use, indications, target groups, clinical claims and
+>   contraindications?
+> - Have the clinical risks and analytical performance/ clinical performance been investigated?
+> - Have relevant MDSW’s characteristics, such as the data input and output, the applied algorithms
+>   or type of interconnection been considered when generating the data to support the performance of
+>   the device?
+> - What is the grade of innovation/ history on the market (how big is the body of scientific evidence)?
+> - Other, as applicable.
+>   Sufficient quality
+> - Were the type and the design of the study/ test appropriate to meet the research objectives?
+> - Was the data set appropriate and actual (state of the art)?
+> - Was the statistical approach appropriate to reach a valid conclusion?
+> - Were all ethical, legal and regulatory considerations/ requirements taken into account?
+> - Is there any conflict of interest?
+
```

Comment:
Does this list need to be answered specifically for each validation? or is having a standardized way for complying with this list enough? Like, could we add a clause that the authorizer of the validation report makes sure that it complies with these requirements? I'm not a fan of having a table with hard to answer questions as a part of the validation template.

Reply:
@TLeenders  this is a quote from the MDCG, I see this as possible things to check. Do we need an additional line to make this more clear??

Reply:
added this: Note that, this guidance helps with Chapter VI of the IVDR, which is not mentioned in article 5.5 for in house developed tests, but could give guidance to define the extend of your validation.